src/modules/addComment.js

```diff
@@ -0,0 +1,33 @@
+import getcomments from './getComments.js';
+import displayComments from './displayComments.js';
+
+const invApiComment = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/6jTH3jOos7OpnHxCg6lz/comments/';
+
+export default (button, username, text, bookId) => {
+  button.addEventListener('click', async (e) => {
+    e.preventDefault();
+    bookId = bookId.toString();
```

Comment:
One of the best javascript capstone projects I reviewed.
Such a clean code that you followed all es6 javascript best practices.

### Keep up the excellent work. 💪

Reply:
@Omar-Muhamad Thanks for the feedback.